docs/endpoints.rst

```diff
@@ -213,6 +213,40 @@ Native container support (docker, singularity, shifter):
 * Soft routing: When receiving a task for a specific container type, the Globus Compute endpoint attempts to send the task to a manager that has a suitable warm container to minimize the total number of container cold starts. If there are not any warmed containers in any connected managers, the Globus Compute endpoint chooses one manager randomly to dispatch the task.
 
 
+Restarting endpoint when machine restarts
+-----------------------------------------
+
+To ensure that a compute endpoint comes back online when its host machine restarts, a
+systemd service can be configured to run the endpoint.
+
+1. Create a service file at ``/etc/systemd/<service-name>.service``
+
+2. Populate that file with the following settings:
+
+.. code-block:: cfg
+
+  [Unit]
+  Description=<service description>
+  After=network.target
+  StartLimitIntervalSec=0
+
+  [Service]
+  ExecStart=<full path to globus-compute-endpoint executable> start <endpoint name>
+  User=<user account>
+  Type=simple
+  Restart=always
+  RestartSec=1
+
+  [Install]
+  WantedBy=multi-user.target
+
+3. Enable the service:
```

Comment:
We may want to add a step to reload the systemd config: `sudo systemctl daemon-reload`. This is technically not required for new service files, but is required for changes to existing service files.

Reply:
In cases where you need to run `sudo systemctl daemon-reload`, do you also need to run `sudo systemctl enable my-globus-compute-endpoint.service`?

Reply:
They have separate functionalities. The former is to make systemd aware of changes to a service file. The latter is to make the service start on boot.